### PR TITLE
Added support for aten::norm.ScalarOpt_dim

### PR DESCRIPTION
**Pull request by GlebKazantaev**
Added support for `aten::norm.ScalarOpt_dim` which is required by `torch.nn.functional.normalize`.

### CODE REVIEW

**Review by ke1337 — APPROVED** [review]

**Review by antoniojkim — APPROVED** [review]

**Comment by GlebKazantaev** [comment]
@silvasean is it okay if I create github issue to cover https://github.com/llvm/torch-mlir/pull/1774/files#diff-882c555d6fd90bc6e098e9d278f623fc33be59f4514eb7d4fd2c294c59a9dca3R11 ? 
I tried to handle torch to linalg conversion for `AtenNormScalarOptDimOp` but looks like it require smth more than just shape inference and type propagation.

**Comment by GlebKazantaev** [comment]
Created an issue: https://github.com/llvm/torch-mlir/issues/1788
